common/test/acceptance/pages/lms/courseware.py

```diff
@@ -128,6 +128,33 @@ def start_proctored_exam(self):
         # elf.wait_for_element_presence(".proctored-exam-code", "unique exam code")
 
     @property
+    def entrance_exam_status_msg(self):
```

Comment:
nit: this is returning the selector but not the message text. can we update the name ?